.github/ISSUE_TEMPLATE/feature-request---proposal.md

```diff
@@ -19,17 +19,17 @@ A clear and concise description of what you want to happen.
 
 A breaking change would require consumers or implementors of the API to modify their code for it to continue to function (ex: renaming of a required field or the change in data type of an existing field). A non-breaking change would allow existing code to continue to function (ex: addition of an optional field or the creation of a new optional endpoint). 
 
-* [ ] Yes, breaking
-* [ ] No, not breaking
-* [ ] I'm not sure
+ [ ] Yes, breaking
+ [ ] No, not breaking
+ [ ] I'm not sure
 
```

Comment:
Maybe remove the `[ ]` part instead of the `*` to maintain the list formatting. Agreed the checkboxes are annoying!